Assignment4/Assignment4.js

```diff
@@ -1,21 +1,31 @@
 const fs = require('fs');
 
-function readAppendWrite (readFile, appendFile, writeFile) {
-  fs.readFile (readFile, function (err1, data1) {
-    if (err1) throw err1;
-    console.log(data1.toString())
-    console.log("----------------------------------------------");
+function checkError(error){
```

Comment:
@aarushi121 We can make only function for error-throwing & printing console messages. Think about it.